theano/tensor/nnet/tests/test_conv3d2d.py

```diff
@@ -153,7 +152,6 @@ def test_conv3d(mode=mode_without_gpu, shared=theano.tensor._shared):
     newconv3d = theano.function([], [],
                                 updates={s_output: out},
                                 mode=mode)
-    check_diagonal_subtensor_view_traces(newconv3d)
```

Comment:
From the comment that says "patched implementation ... with Tf=1", I guess it is normal that these Ops are not there, but I'll have a quick look.